<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasternodeCreateDialog</class>
 <widget class="QDialog" name="MasternodeCreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>298</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>657</width>
    <height>298</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>ZIXX - Create a masternode wizard</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="horizontalSpacing">
    <number>12</number>
   </property>
   <property name="verticalSpacing">
    <number>9</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="verticalTabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <widget class="QWidget" name="tabOverview">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Overview</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="overviewHorizontalLayout" stretch="1,1">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <widget class="QLabel" name="overviewStatusLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>This wizard will guide you through he process of creating a hot masternode!
Before we begin, however, we must check to verify if your wallet fulfills the necessary pre-requisites.
To operate a Masternode, you need to have a balance of 1000 XZX on your wallet, as well as being able to have the correct port (TCP:44845) open on your firewall, making your device connectable.
Press the &quot;Continue&quot; button to evaluate the conditions and proceed.</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
           <property name="indent">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkboxFundsAvailable">
             <property name="text">
              <string>Funds Available</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QCheckBox" name="checkboxFundsConfirmed">
             <property name="text">
              <string>Funds Confirmed</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="checkboxIPAddressValid">
             <property name="text">
              <string>Connectivity Status</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="sizeConstraint">
              <enum>QLayout::SetFixedSize</enum>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btClose">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777214</height>
                </size>
               </property>
               <property name="text">
                <string>Close</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btContinue">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Continue</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QProgressBar" name="progressBar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="errorLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkboxWalletLocked">
             <property name="text">
              <string>Wallet Unlocked</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
